pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>4.2</version>
+        <relativePath />
+    </parent>
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>manage-permission</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.222</jenkins.version>
+        <java.level>8</java.level>
+        <useBeta>true</useBeta>
+    </properties>
+    <name>Manage permission plugin</name>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+    <!--TODO the goal is to use README as plugin documentation, url to be validated once in jenkinci organization-->
+    <url>https://github.com/jenkinsci/manage-permission-plugin</url>
+    <developers>
+        <developer>
+            <id>amuniz</id>
+            <url>https://github.com/amuniz</url>
+        </developer>
+        <developer>
+            <id>mikecirioli</id>
+            <url>https://github.com/mikecirioli</url>
+        </developer>
+        <developer>
+            <id>escoem</id>
+            <url>https://github.com/escoem/</url>
+        </developer>
+        <developer>
+            <id>EstherAF</id>
+            <url>https://github.com/EstherAF</url>
+        </developer>
+        <developer>
+            <id>aHenryJard</id>
+            <url>https://github.com/aHenryJard</url>
+        </developer>
+    </developers>
+    <!-- Assuming you want to host on @jenkinsci:
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+    <scm>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+    </scm>
+    -->
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </pluginRepository>
+    </pluginRepositories>
```

Comment:
`repositories` and `pluginRepositories` comes [from the parent](https://github.com/jenkinsci/plugin-pom/blob/88cb710c1b7244bf5f885e16fc9f0baef1ef2661/pom.xml#L851-L863). It can be removed.